### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'html'

### ISSUE THREAD

**Issue by w062611** [kevinzg/facebook-scraper#15]
Hi there,

Thanks for your great efforts. It's great to see this kind of package since FB changed its API policy last year.  

I was trying to leverage your code to collect some data. The following is my code. However, when I executed my code, I got the "AttributeError: 'NoneType' object has no attribute 'html'. Do you know how to fix the issue? 

d = []
for post in get_posts('Chrysler', pages=1000):
    d.append({'time': post['time'], 'post_id': post['post_id'], 'text': post['text'][:20000], 'like':
       post['likes'], 'comment':post['comments'],'share': post['shares'] , 'URL':post['post_url'], 'link':post['link']})
    
Obj1= pd.DataFrame(d)
Obj1.to_csv("fgc_Chrysler.csv")
    
Thank a lot!

**Comment by hanubence** [comment]
I've encountered the same error, and posts shared from another page seem to be causing them, however I haven't yet found a way to skip those posts (I'm new to python so I hope the pros will figure this out).

Anyways I was able to collect the data I needed so I'm thankful for the developers.

**Comment by hanubence** [comment]
Alright so I figured out this "fix", it's a terrible solution, but at least it makes it work.

Just add that if line in facebook_scraper.py around line 212 which checks if there is actually a container or not, which there is not whenever you're getting a shared post.

![image](https://user-images.githubusercontent.com/32911312/68085696-049add00-fe44-11e9-9bc9-4fdcbe0db6e3.png)

Again, this is a terrible solution, but I just needed to scrape 1 batch of data, so this works for me.

The updated method (though code formatting doesn't seem to work for me in github):
`def _find_and_search(article, selector, pattern, cast=str):
    container = article.find(selector, first=True)
    if container is not None:
        match = pattern.search(container.html)
        return match and cast(match.groups()[0])`
